examples/polish_taxes/tests/test_a7_u1_p1.catala_pl

```diff
@@ -0,0 +1,23 @@
+> Include: ../podatek_od_czynnosci_cywilnoprawnych/rozdzial_3.catala_pl
+
+## [Test - Art. 7 ustęp 1 punkt 1]
+
+```catala
+deklaracja zakres Test_A7_U1_P1_PPa:
+  kontekst sprzedaz zakres UmowaSprzedazy
+
+zakres Test_A7_U1_P1_PPa:
+  definicja sprzedaz.kwota wynosi $100
+
+  asercja sprzedaz.podatek = $2
+
+
+deklaracja zakres Test_A7_U1_P1_PPb:
+  kontekst sprzedaz zakres UmowaSprzedazy
+
+zakres Test_A7_U1_P1_PPb:
+  definicja sprzedaz.kwota wynosi $100
+  definicja sprzedaz.inne_prawa_majatkowe wynosi prawda
+
+  asercja sprzedaz.podatek = $1
```

Comment:
By tweaking the regex in the lexer, you can customize the money amount to use the Polish monetary sign and the Polish number writing convention.